neo/Ledger/Blockchain.cs

```diff
@@ -27,17 +27,17 @@ public class ImportCompleted { }
         public class FillMemoryPool { public IEnumerable<Transaction> Transactions; }
         public class FillCompleted { }
 
-        public static readonly uint SecondsPerBlock = ProtocolSettings.Default.SecondsPerBlock;
+        public static readonly uint MillisecondsPerBlock = ProtocolSettings.Default.MillisecondsPerBlock;
```

Comment:
ulong

Reply:
Here it is ok, @shargon. It can be uint, because we just dealing with milliseconds.
The other one is the sum of milliseconds.

But if you think that, for consistency, it is better ulong we can change it.

Reply:
Then is ok

Reply:
Perfect, lets make it faster...whauah